src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,107 @@
+import { getNumbers } from '../../utils';
+
+interface Props {
+  items: string[];
+  ItemsPerPage: number;
+  currentPage: number;
+  setCurrentPage: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  ItemsPerPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  function creativePageContent(
+    data: string[],
+    listLegth: number,
```

Comment:
There is a typo in the parameter name `listLegth`. It should be `listLength` to correctly represent the length of the list.